packages/eslint-config/index.js

```diff
@@ -19,9 +20,9 @@ module.exports = {
   settings: {
```

Comment:
I think this whole section can go away as node_module is the default resolver folder anyways?